cmd/dashboard/main.go

```diff
@@ -49,6 +49,7 @@ var (
 	logoutUrl          = flag.String("logout-url", "", "If set, enables logout on the frontend and binds the logout button to this url")
 	csrfSecureCookie   = flag.Bool("csrf-secure-cookie", true, "Enable or disable Secure attribute on the CSRF cookie")
 	tenantNamespace    = flag.String("namespace", "", "If set, limits the scope of resources watched to this namespace only")
+	loggerPreset       = flag.String("logger-preset", "production", "Logger preset (production, example or development)")
```

Comment:
can we hide the implementation details here and go with something like: "--loglevel: debug" instead?
if we switch our logger package in future I'd rather not be tied to zap's preset names

Reply:
Sure, i liked the preset idea as it's simple but ok for log level, maybe combined with a log format (json or console), structured logging is nice when collecting logs but a bit painful when reading in the console.

Reply:
yep that sounds reasonable

Reply:
i choose either production or develpment preset according to the format:
- json -> production
- console -> development

Reply:
done